modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -5809,8 +5809,7 @@ void GDScriptAnalyzer::validate_call_arg(const List<GDScriptParser::DataType> &p
 #ifdef DEBUG_ENABLED
 void GDScriptAnalyzer::is_shadowing(GDScriptParser::IdentifierNode *p_identifier, const String &p_context, const bool p_in_local_scope) {
 	const StringName &name = p_identifier->name;
-	GDScriptParser::DataType base = parser->current_class->get_datatype();
-	GDScriptParser::ClassNode *base_class = base.class_type;
+	const GDScriptParser::DataType current_class_type = parser->current_class->get_datatype();
```

Comment:
Nitpick: this isn't used until line 5841. It's recommended to declare variables as close to the first use as possible, if it doesn't affect anything.

Reply:
I added it above:
```
if (p_in_local_scope) {
	GDScriptParser::ClassNode *base_class = current_class_type.class_type;
```
this way it is accessible for `base_class` and `native_base_class` (defined later).